core/swapper.py

```diff
@@ -41,7 +41,7 @@ def process_faces(source_face, frame, progress, all_faces=False):
     return frame
 
 
-def process_video(source_img, frame_paths):
+def process_video(source_img, frame_paths, process_frame):
```

Comment:
process_frame is a callback that you fire? lets call it that way then... preview_callback